n_helpers.c

```diff
@@ -263,13 +425,13 @@ const char * NoteBinaryReceive(uint8_t * buffer, size_t bufLen,
     --bufLen;
 
     uint32_t decLen = bufLen;
-    // Decode it in-place, which is safe because decoding shrinks
+    // Decode it in place, which is safe because decoding shrinks
```

Comment:
in-place is right: https://en.wikipedia.org/wiki/In-place_algorithm

Reply:
not here.